apps/site-nft/src/components/AddAccount/Store/Providers/Google/StoreGoogle.tsx

```diff
@@ -157,7 +156,7 @@ function fetchAndVerifyWallet(address: string) {
     alert("Cannot upload this wallet: it is not a local account");
     throw new Error("Could not find local account: " + address);
   }
-  if (wallet.privateKey) {
+  else if (wallet.privateKey) {
     alert("Could upload decrypted wallet - if you are seeing this, contact support@thecoin.io");
```

Comment:
We should take the time to change that in a translatable message.

Reply:
Issue created to remind me.  It should be done, but it's not central to this PR (and I need these PR's in tonight so can setup testing thingy tomorrow)